Hackney.Shared.HousingSearch/Domain/Tenure/TemporaryAccommodationInfo.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Linq;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using Hackney.Shared.Tenure.Domain;
+
+namespace Hackney.Shared.HousingSearch.Domain.Tenure
+{
+    public class TemporaryAccommodationInfo
+    {
+        public string BookingStatus { get; set; }
+
+        public static TemporaryAccommodationInfo Create(QueryableTemporaryAccommodationInfo temporaryAccommodationInfo)
```

Comment:
Would be good to cover this with a test

Reply:
Addressed in [c957eac](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/c957eacde0b3cc200e725b85610f4e0e8f78760b).